client/blocks/reader-full-post/style.scss

```diff
@@ -7,50 +7,114 @@
 }
 
 .reader-full-post__content {
+
 	@include breakpoint( ">1280px" ) {
 		width: 720px;
 		padding-left: 260px;
 	}
+
 	@include breakpoint( "1040px-1280px" ) {
 		width: 720px;
 		padding-left: 240px;
 	}
+
 	@include breakpoint( "660px-1040px" ) {
 		width: auto;
 		margin: 0;
 		padding-left: 240px;
 	}
+
 	@include breakpoint( "<660px" ) {
 		width: auto;
 		padding: 0 20px;
 	}
+
 	margin: 0 auto;
 }
 
 .reader-full-post__back-container {
 	margin: 0;
 	position: fixed;
 		top: 47px;
-		left: 0px;
+		left: 0;
 	z-index: z-index( '.masterbar', '.reader-back'	);
 
 	.button.is-compact.is-borderless {
 		padding: 18px 40px 18px 20px;
+
+		@include breakpoint( "<660px" ) {
+			padding: 20px 8px 16px;
+		}
 	}
 
 	@include breakpoint( "<660px" ) {
 		top: 0;
 		left: 0;
 		right: 0;
-		height:46px;
-		background: #fff;
+		height: 46px;
+		background: $white;
 		border-bottom: 1px solid $gray-light;
-		margin-top: 0;
+	}
+}
+
+.reader-full-post__back-container .reader-full-post__back-label {
+
+	@include breakpoint( "<660px") {
+		display: none;
+	}
+}
+
+.reader-full-post__visit-site-container {
+	font-size: 11px;
+	font-weight: 500;
+	margin: 0;
+	position: fixed;
+		right: 0;
+		top: 47px;
+	text-transform: uppercase;
+	z-index: z-index( '.masterbar', '.reader-visit-site' );
+
+	.external-link .gridicons-external {
+		fill: darken( $gray, 10% );
+		top: 3px;
+
+		@include breakpoint( "<660px" ) {
+			top: 6px;
+		}
+	}
+
+	.external-link {
+		color: darken( $gray, 10% );
+		display: block;
+		padding: 10px 20px 15px 20px;
+
+		@include breakpoint( "<660px" ) {
+			padding: 10px 10px 15px 6px;
+		}
 
-		.button.is-compact.is-borderless {
-			padding: 20px 40px 16px 20px;
+		&:hover {
+			color: $gray-dark;
+
+			.gridicons-external {
+				fill: $gray-dark;
+			}
 		}
 	}
+
+	.reader-full-post__visit-site-label {
+
+		@include breakpoint( "<660px" ) {
+			display: none;
```

Comment:
I don't think mobile-first applies here since that would mean I'd have to add an additional `display` declaration outside of `<660px` which wouldn't save us code. On Devdocs it says:

> There is only one time when using a < breakpoint is ok; when it saves you code: